web-dashboard/index.html

```
@@ -0,0 +1,1398 @@
+      <span class="term-line"> </span>
+    </div>
+    <div class="quick-cmds" style="flex-direction:column;gap:6px;">
+      <div style="display:flex;flex-wrap:wrap;gap:5px;padding-bottom:4px;border-bottom:1px solid var(--surface0);">
+        <span style="font-size:10px;color:var(--overlay0);text-transform:uppercase;letter-spacing:.8px;align-self:center;margin-right:4px;">npm</span>
+        <button class="qbtn" onclick="runCmd('merge')" id="btn-merge">merge</button>
+        <button class="qbtn" onclick="runCmd('verify')" id="btn-verify">verify</button>
+        <button class="qbtn" onclick="runCmd('normalize')" id="btn-normalize">normalize</button>
+        <button class="qbtn" onclick="runCmd('dedup')" id="btn-dedup">dedup</button>
+        <button class="qbtn" onclick="runCmd('doctor')" id="btn-doctor">doctor</button>
+        <button class="qbtn" onclick="runCmd('sync-check')" id="btn-sync-check">sync-check</button>
+        <button class="qbtn" onclick="runCmd('update:check')" id="btn-update-check">update:check</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)